### PR TITLE
fix: rearrange and addition of fields

### PR DESCRIPTION
**Pull request by shekhar16**
https://github.com/GluuFederation/oxTrust/issues/2211

### CODE REVIEW

File: persistence-model/src/main/java/org/oxauth/persistence/model/ClientAttributes.java [hunk 1]
```diff
@@ -55,7 +55,20 @@ public class ClientAttributes implements Serializable {
...
+    private String redirectRegex;
+    
+	@JsonProperty("umaRPTClaimsScripts")
+	private List<String> umaRPTModificationScripts;
+	
+	@JsonProperty("updateTokenScripts")
+	private List<String> updateTokenScripts;
+	
+	@JsonProperty("defaultPromptLogin")
+	private String defaultPromptLogin;
```
Comment (yuriyz) [inline persistence-model/src/main/java/org/oxauth/persistence/model/ClientAttributes.java:70]: These 4 attributes are not supported in oxauth. They are supported in jans only. What is the sense of adding it ? We shoudn't do it.

**Review by yuriyz — CHANGES_REQUESTED** [review]

File: persistence-model/src/main/java/org/oxauth/persistence/model/ClientAttributes.java (hunk 1 above)
Comment (shekhar16) [reply persistence-model/src/main/java/org/oxauth/persistence/model/ClientAttributes.java:70]: @yuriyz  Mike suggested to add these fields.

Comment (yuriyz) [reply persistence-model/src/main/java/org/oxauth/persistence/model/ClientAttributes.java:70]: To oxauth ? It must be some mis-understanding. cc @nynymike

Comment (nynymike) [reply persistence-model/src/main/java/org/oxauth/persistence/model/ClientAttributes.java:70]: Yuriy is right... if they are not in oxAuth they are not needed.

Comment (shekhar16) [reply persistence-model/src/main/java/org/oxauth/persistence/model/ClientAttributes.java:70]: Ok so I m closing this PR.

**Pull request closed without merging**